sessions/recap-project-crud/crud-recap-tourio/README.md

```diff
@@ -0,0 +1,166 @@
+
+- **Integrate the Card Component**
+
+  - Replace the temporary display of place details within your map function in `index.js` with the `PlaceCard` component.
+  - Pass the necessary data (`image`, `name`, and `location`) as props to each `PlaceCard` instance.
+
+- **Whitelisting Image URLs**
+
+  - If not already done so, modify the `PlaceCard` component to use the `Image` component from Next.js for displaying the images. Ensure that you configure your Next.js project to allow images from the URLs you are using by editing the `next.config.js` file and adding the image domains to the `images.domains` array.
+
+### 2. Add new Places
+
+- **Create a New Page for Adding Entries**
+
+  - Add a new file named `create.js` within the `/pages` directory.
+  - This page will host a form that allows users to add new entries to your application.
+  - Add a link within the page that allows users to navigate back to the homepage.
+
+- **Form Component for New Entries**
+
+  - Create a form component that includes fields for `name`, `location`, `mapURL`, `image`, and `description`. You can structure this component in a directory like `components/Form.js`.
+  - Include a submit button in the form.
+  - Basic styling is sufficient.
+
+- **Move State to `_app.js`**
+
+  - To share state across different pages, move the state management for your places (initially in `/pages/index.js`) to `_app.js`. This will make the state accessible throughout your application.
+
+- **Implement the `handleAddPlace` Function**
+
+  - In your `_app.js` write a function named `handleAddPlace` that will be responsible for adding a new place to the state.
+  - This function should accept the form data as an argument, generate a unique ID for the new entry using `uuid`, and then add the new place to the existing state.
+
+- **State and Handler Propagation**
+
+  - Ensure the shared state and the `handleAddPlace` function are properly passed down to the pages and components that require them.
+  - In `create.js`, use these props to manage the form submission and to maintain the list of places.
+
+- **Form Submission and Page Redirection**
+
+  - Upon form submission, use the `handleAddPlace` function to add the new entry to your state.
+  - After adding the entry, redirect the user to the homepage. This can be achieved by importing `useRouter` from `next/router` and calling `router.push('/')` after the form submission logic.
+
+- Link to create page
+  - in `/pages/index.js` add a Link that forwards users to `/pages/create.js`
+  - this link should always be visible
+
+### 3. Implement Detail View for Places
+
+- **Create a Dynamic Route for Place Details**
+
+  - In the `/pages/places` directory, create a dynamic route named `[id]/index.js`. This file will serve as the detail page for each place.
+  - The `[id]` in the file path is a placeholder that Next.js will replace with the actual ID of the place being viewed.
```

Comment:
Fair point!

I will remove the bullet point, since explaining what happens here is already done in the corresponding session and there is no need to have an explanation here (even if it were correct)